common/lib/xmodule/xmodule/modulestore/tests/test_inheritance.py

```diff
@@ -0,0 +1,62 @@
+"""
+Unit tests for testing inheritance mixins
+"""
+from __future__ import absolute_import
+
+import ddt
+import unittest
+
+from mock import Mock
+from django.utils.timezone import now, timedelta
+from xblock.core import XBlock
+from xblock.fields import ScopeIds
+from xblock.runtime import DictKeyValueStore, KvsFieldData, Runtime
```

Comment:
to remove `DictKeyValueStore, KvsFieldData` as they are not being used in tests anymore.